episodes/4-advanced-layer-types.Rmd

```diff
@@ -267,6 +270,8 @@ We have 100 matrices with 3 * 3 * 3 = 27 values each so that gives 27 * 100 = 27
 ::::
 :::
 
+## 4. Choose a pretrained model or start building architecture from scratch
```

Comment:
This might be better suited to be moved above before Convolution layers are introduced? Since the layer type has to do more with the architecture than data prep.

Reply:
Totally agree! I moved it :)